sympy/series/tests/test_limits.py

```diff
@@ -640,6 +640,11 @@ def test_issue_10801():
     assert limit(16**k / (k * binomial(2*k, k)**2), k, oo) == pi
 
 
+def test_issue_10868():
+    assert limit(log(x)+asech(x),x,0, dir = '+-') == log(2)
```

Comment:
Spaces after commas, no space around = in `dir = `keep the formatting consistent and follow PEP8